content/en/blog/_posts/2022-04-19-csi-nodeexpandsecret.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog
+title: >-
+  Kubernetes 1.25: Use Secrets for Node-Driven Expansion of CSI Volumes
+date: 2022-08-28T10:00:00-08:00
```

Comment:
In 420 blog posts, only 6 of them use `2022-08-02T12:00:00+08:00` format.
412 of them using `date: 2022-08-02` format.

```suggestion
date: 2022-08-28
```